configure.ac

```diff
@@ -47,9 +47,9 @@ AC_SUBST([WOLFSSL_CONFIG_ARGS])
 # The three numbers in the libwolfssl.so.*.*.* file name. Unfortunately
 # these numbers don't always line up nicely with the library version.
 WOLFSSL_LIBRARY_VERSION_FIRST=42
-WOLFSSL_LIBRARY_VERSION_SECOND=0
+WOLFSSL_LIBRARY_VERSION_SECOND=1
 WOLFSSL_LIBRARY_VERSION_THIRD=0
-WOLFSSL_LIBRARY_VERSION=42:0:0
+WOLFSSL_LIBRARY_VERSION=43:0:1
```

Comment:
@douzzer does this make sense to you?

Reply:
Nope I'm not getting it either -- AFAIK the first field in `WOLFSSL_LIBRARY_VERSION` should match `WOLFSSL_LIBRARY_VERSION_FIRST`.

Reply:
@dgarske @JacobBarthelmeh This awkward inconsistency is because we're using `-version-info` instead of `-version-number` in `src/include.am`.  The weird `43:0:1` value is to invert the arithmetic that `libtool` applies internally to its `-version-info` arg.

We should have
```
src_libwolfssl@LIBSUFFIX@_la_LDFLAGS = ${AM_LDFLAGS} -no-undefined -version-number ${WOLFSSL_LIBRARY_VERSION}
```

Then `WOLFSSL_LIBRARY_VERSION` and the `_FIRST` `_SECOND` `_THIRD` variables will stay locked together permanently as one would expect.

In fact then we can have
```
WOLFSSL_LIBRARY_VERSION=${WOLFSSL_LIBRARY_VERSION_FIRST}:${WOLFSSL_LIBRARY_VERSION_SECOND}:${WOLFSSL_LIBRARY_VERSION_THIRD}
```